### PR TITLE
feat(ios): expose api to use sf images

### PR DESCRIPTION
**Pull request by vijaysingh-axway**
https://jira.appcelerator.org/browse/TIMOB-27133

### CODE REVIEW

File: apidoc/Titanium/UI/iOS/iOS.yml [hunk 1]
```diff
@@ -40,6 +40,21 @@ methods:
     since: "5.2.0"
     osver: {ios: {min: "9.1"}}
 
+  - name: systemImage
+    summary: |
+        Get image from SF Symbol provided by Apple.
+    description: |
+        More details on SF Symbols can be found from https://developer.apple.com/design/human-interface-guidelines/sf-symbols/overview.
```
Comment (janvennemann) [inline apidoc/Titanium/UI/iOS/iOS.yml]:
```suggestion
        More details on SF Symbols can be found on https://developer.apple.com/design/human-interface-guidelines/sf-symbols/overview.
```

File: apidoc/Titanium/UI/iOS/iOS.yml [hunk 2]
```diff
@@ -40,6 +40,21 @@ methods:
     since: "5.2.0"
     osver: {ios: {min: "9.1"}}
 
+  - name: systemImage
+    summary: |
+        Get image from SF Symbol provided by Apple.
```
Comment (janvennemann) [inline apidoc/Titanium/UI/iOS/iOS.yml]:
```suggestion
        Get image from SF Symbols provided by Apple.
```

File: apidoc/Titanium/UI/iOS/iOS.yml [hunk 3]
```diff
@@ -40,6 +40,21 @@ methods:
...
+    description: |
+        More details on SF Symbols can be found from https://developer.apple.com/design/human-interface-guidelines/sf-symbols/overview.
+        One can download SF Symbols app from https://developer.apple.com/design/downloads/SF-Symbols.dmg and 
+        can get the inbuilt SF Symbol name.
```
Comment (janvennemann) [inline apidoc/Titanium/UI/iOS/iOS.yml]:
```suggestion
        use it to lookup the inbuilt SF Symbol name.
```

**Review by janvennemann — CHANGES_REQUESTED** [review]

**Review by janvennemann — APPROVED** [review]
LGTM!

**Comment by janvennemann** [comment]
@vijaysingh-axway please add a unit test for the new api, thanks!

**Review by ssjsamir — APPROVED** [review]
FR Passed: Able to make use of sf images, using the following test case: 
```
var win = Ti.UI.createWindow({
backgroundColor: '#fff'
});
var triangle = Ti.UI.iOS.systemImage('drop.triangle.fill');
var forward = Ti.UI.iOS.systemImage('forward');
var backward = Ti.UI.iOS.systemImage('backward');
var folder = Ti.UI.iOS.systemImage('folder.fill');
 
 
var imageView = Ti.UI.createImageView({
width: '150',
height: '150',
image: triangle,
tintColor: 'red'
});
 
 
var button = Ti.UI.createButton({
top: 60,
width: 100,
height: 50,
//backgroundImage: folder
image: folder
})
 
var stepper = Ti.UI.iOS.createStepper({
top: 150,
incrementImage: forward,
decrementImage: backward
});
 
 
win.add(button);
win.add(stepper);
win.add(imageView);
win.open();
```

**Test Environment**
```
MacOS Mojave version 10.14.4
Xcode 11 beta 5
Node.js ^8.11.1
iPhone 8 (13.0 Sim)
"NPM":"4.2.14","CLI":"7.1.1-master.2"
```

**Comment by hansemannn** [comment]
Can this be merged please?

**Comment by hansemannn** [comment]
@vijaysingh-axway @janvennemann @ssjsamir There was an issue with device builds, since the `TiBlob` class was not imported properly. This is fixed via https://github.com/vijaysingh-axway/titanium_mobile/pull/4 which has Vijays fork as the head, so once merged it should appear here.

**Comment by vijaysingh-axway** [comment]
Thanks @hansemannn . Merged.

**Comment by hansemannn** [comment]
We also see issues with this when using it in conditional code, even with Alloy.Globals constants. Can you reproduce it in Alloy?

**Pull request merged**